src/com/mambu/apisdk/services/LoansService.java

```diff
@@ -1019,28 +1019,73 @@ public LoanAccount updateLoanAccountGuarantees(String accountId, List<Guaranty>
 	/**
 	 * Get loan transactions by specifying filter constraints
 	 * 
+	 * Note: This method is deprecated, you may use the getLoanTransactionsWithFullDetails in order to obtain loan
+	 * transaction with full details (custom field included) or getSavingsAccountsWithBasicDetails to obtain the loan

```

Comment:
it should be "or getLoanTransactionsWithBasicDetails" instead of "or getSavingsAccountsWithBasicDetails"

Reply:
also, custom field**s**

Reply:
Done, thanks!